references/detection/group_by_aspect_ratio.py

```diff
@@ -101,7 +101,7 @@ def __len__(self):
         collate_fn=lambda x: x[0])
     aspect_ratios = []
     with tqdm(total=len(dataset)) as pbar:
-        for i, (img, _) in enumerate(data_loader):
+        for _i, (img, _) in enumerate(data_loader):
             pbar.update(1)
             height, width = img.shape[-2:]
             aspect_ratio = float(height) / float(width)
```

Comment:
@fmassa Shouldn't `aspect_ratio` be `width/height`?

Reply:
I think the important thing in here is for it to be consistent between all the implementations, and I don't think it will change anything in this particular case.

Reply:
Well, it just looks weid to understand because it is different from other definition.

Reply:
which definition?

Reply:
Something like [this](https://en.wikipedia.org/wiki/Aspect_ratio) and [this](https://techterms.com/definition/aspect_ratio), and also `aspect_ratio = w / h` is commonly defined in other places like computer graphics. Anyway, it doesn't affect the behavior in your code.

Reply:
I believe we can change this definition here without breaking the rest of the code. I would be willing to merge a PR that do this change.

Reply:
@fmassa Please take a look at #1194